src/GrpAb/Map.jl

```diff
@@ -216,6 +216,8 @@ end
 
 ==(f::FinGenAbGroupHom, g::FinGenAbGroupHom) = domain(f) === domain(g) && codomain(f) === codomain(g) && all(x -> f(x) == g(x), gens(domain(f)))
 
+Base.hash(f::FinGenAbGroupHom, h::UInt) = zero(UInt)
```

Comment:
```suggestion
Base.hash(f::FinGenAbGroupHom, h::UInt) = h
```
this is better for compositions of hash functions